views/js/qtiCreator/model/helper/container.js

```diff
@@ -21,8 +21,13 @@ define(['jquery', 'lodash', 'taoQtiItem/qtiCreator/model/qtiClasses'], function(
 
                 //register and name all loaded classes:
                 var Qti = {};
-                for(var i in arguments){
-                    Qti[arguments[i].prototype.qtiClass] = arguments[i];
+                for(var i in arguments) {
```

Comment:
because of the special nature of `arguments` (it's an iterable), I would rather use something more secure, like (not tested but the idea is there):
```js
var Qti = _.reduce([].slice.call(arguments), function(acc, qtiClassElt){
      acc[qtiClassElt.prototype.qtiClass] = qtiClassElt;
      return acc;
}, {})
```
The important part is to transform `arguments` into an array using `[].slice.call(arguments)`